packages/components/src/Button/button.stories.tsx

```diff
@@ -101,3 +139,79 @@ withFakeClassName.args = {
   variant: "outline",
   className: "fake-className",
 };
+
+// Show grids with all variants
+
+const gridParameters = {
+  axe: {
+    skip: true,
+  },
+  snapshot: {
+    skip: true,
+  },
+};
+
+const renderSize = (
+  size: ButtonProps["size"],
+  textColor: "white" | "neutral",
+  children: React.ReactNode,
+) =>
+  variants.map((variant) => {
+    // For now, the UI kit only includes alert & brand "flat" buttons
+    const colors = variant === "flat" ? ["alert", "brand"] : allColors;
+
+    return (
+      <React.Fragment key={variant}>
+        <Heading size="h2" color={textColor}>
+          {variant} - {size}
+        </Heading>
+        <table className={styles["variant"]}>
```

Comment:
Really doesn't matter, but I find it easier to type

```suggestion
        <table className={styles.variant}>
```